bindings/nodejs/hawktracer_client_nodejs.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef HAWKTRACER_CLIENT_NODEJS_HPP
+#define HAWKTRACER_CLIENT_NODEJS_HPP
+
+#include "client_context.hpp"
+
+#include <napi.h>
+
+namespace HawkTracer
+{
+namespace Nodejs
+{
+
+using namespace Napi;
+
+class Client: public ObjectWrap<Client>
+{
+public:
+    static Object init_bindings(class Env env, Object exports);
+
+    explicit Client(const CallbackInfo &info);
+    ~Client() override;
+
+    class Value start(const CallbackInfo &info);
+    void stop(const CallbackInfo &info);
+    void set_on_events(const CallbackInfo &info);
+
+private:
+    void _stop();
+    std::unique_ptr<std::vector<parser::Event>>
+    handle_events(std::unique_ptr<std::vector<parser::Event>> events, ClientContext::ConsumeMode consume_mode);
+    static class Value convert_field_value(class Env env, const parser::Event::Value &value);
```

Comment:
```suggestion
    static Value _convert_field_value(class Env env, const parser::Event::Value &value);
```

Reply:
The compiler insists on using `class Value` and `class Env` probably because part of the declarations are done in an internal header (napi-inl.h).

Reply:
Renamed in [4f074](https://github.com/amzn/hawktracer/pull/67/commits/4f07400dc79d7288fe687268ec6b6c2b87908d85)